wolfcrypt/src/aes.c

```diff
@@ -10833,18 +10970,19 @@ static int wc_AesFeedbackCFB8(Aes* aes, byte* out, const byte* in,
 
     RESTORE_VECTOR_REGISTERS();
 
-    return 0;
+    return ret;
 }
 
 
 /* returns 0 on success and negative values on failure */
-static int wc_AesFeedbackCFB1(Aes* aes, byte* out, const byte* in,
+static WARN_UNUSED_RESULT int wc_AesFeedbackCFB1(Aes* aes, byte* out, const byte* in,
```

Comment:
Why is `wc_AesFeedbackCFB1` adding `WARN_UNUSED_RESULT`? It already had `int` return type.

Reply:
I added `WARN_UNUSED_RESULT` to _all_ `static` non-`void` funcs in aes.c, opportunistically.  catches usage errors now and forever.